sechub-pds-solutions/loc/docker/scripts/scc.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+echo "Running PDS Scc"
+
+extracted_folder=""
+if [[ "$PDS_JOB_HAS_EXTRACTED_SOURCES" == "true" ]]
+then
+    echo "Found extracted sources"
+    extracted_folder="$PDS_JOB_EXTRACTED_SOURCES_FOLDER"
+else
+    echo ""
+    echo "ERROR: No source code."
+    exit 1
+fi
+
+# Call scc and convert Yaml to Json
+# We need to turn the "version" attribute to "cloc_version" so the SecHub server will parse it.
```

Comment:
Please create an issue on GitHub for this. Maybe, the parser for the loc component can use another field/combination to recognize the report format.